source/elements/oneDPL/source/parallel_api/iterators.rst

```diff
@@ -64,6 +86,8 @@ counter; dereference operations cannot be used to modify the counter. The arithm
 operators of ``counting_iterator`` behave as if applied to the values of Integral type
 representing the counters of the iterator instances passed to the operators.
 
+``counting_iterator`` are SYCL device-copyable, and are "device accessible content iterators".
```

Comment:
I was fine with keeping "always for `counting_iterator` and `discard_iterator` (I think I wrote something like "iterator below". But how it's right now is also good. Please keep it as is.

I have another suggestion somewhere you use "`<iterator_name>` are" and somewhere "`<iterator_name>` is". I propose to align to "is" everywhere
```suggestion
``counting_iterator`` is SYCL device-copyable, and are "device accessible content iterators".
```

Reply:
adjusted all of these "are" -> "is" [a]